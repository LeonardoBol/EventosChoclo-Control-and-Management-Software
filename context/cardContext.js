import { createContext, useState } from 'react';

export const cardContext = createContext();

export const CardProvider = ({ children }) => {
    const [eventos, setEventos] = useState([]);
    return (
        <cardContext.Provider value={
            {eventos,
            setEventos}
        }>
            { children}
        </cardContext.Provider>
    )
}
