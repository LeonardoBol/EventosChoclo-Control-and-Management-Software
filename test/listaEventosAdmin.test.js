import { configure, mount } from 'enzyme';
import adapter from 'enzyme-adapter-react-17-updated';
import Carrusel from '../components/listarEventos/carrusel';
import { CardProvider } from '../context/cardContext';
/* import ListarEventos from '../pages/eventosAdmin/listarEventos'; */

//Cristian Medina 1824150
configure({
    adapter: new adapter()
})

let wrapper = mount(
    <CardProvider>
        <Carrusel />
    </CardProvider>
)

describe('Probando Renderizado de cardEventos', () => {
    test('Deben aparecer todas las cardEventos ', () => {
        expect(wrapper.html()).toMatchSnapshot();
    })
})
