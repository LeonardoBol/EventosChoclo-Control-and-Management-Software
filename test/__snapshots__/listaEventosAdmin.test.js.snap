// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando Renderizado de cardEventos Deben aparecer todas las cardEventos  1`] = `"<div class=\\"carruselEventoAdmin\\"><div class=\\"carruselHeader\\"><h1>Eventos Actuales</h1><h1>Crear Evento</h1></div><div class=\\"ant-carousel\\"><div class=\\"slick-slider slick-initialized\\" dir=\\"ltr\\" style=\\"width: 900px;\\"><div class=\\"slick-list\\"><div class=\\"slick-track\\" style=\\"opacity: 1; transform: translate3d(0px, 0px, 0px);\\"><div data-index=\\"-1\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><div tabindex=\\"-1\\" style=\\"width: 100%; display: inline-block;\\"><div style=\\"height: 600px;\\">2</div></div></div></div><div data-index=\\"0\\" class=\\"slick-slide slick-active slick-current\\" tabindex=\\"-1\\" aria-hidden=\\"false\\" style=\\"outline: none; width: 0px;\\"><div><div class=\\"listEventos\\" tabindex=\\"-1\\" style=\\"width: 100%; display: inline-block;\\"><div class=\\"list\\"><div class=\\"eventoCard\\"><div class=\\"ant-skeleton ant-skeleton-active\\"><div class=\\"ant-skeleton-content\\"><h3 class=\\"ant-skeleton-title\\" style=\\"width: 38%;\\"></h3><ul class=\\"ant-skeleton-paragraph\\"><li></li><li></li><li style=\\"width: 61%;\\"></li></ul></div></div></div><div class=\\"eventoCard\\"><div class=\\"ant-skeleton ant-skeleton-active\\"><div class=\\"ant-skeleton-content\\"><h3 class=\\"ant-skeleton-title\\" style=\\"width: 38%;\\"></h3><ul class=\\"ant-skeleton-paragraph\\"><li></li><li></li><li style=\\"width: 61%;\\"></li></ul></div></div></div><div class=\\"eventoCard\\"><div class=\\"ant-skeleton ant-skeleton-active\\"><div class=\\"ant-skeleton-content\\"><h3 class=\\"ant-skeleton-title\\" style=\\"width: 38%;\\"></h3><ul class=\\"ant-skeleton-paragraph\\"><li></li><li></li><li style=\\"width: 61%;\\"></li></ul></div></div></div><div class=\\"eventoCard\\"><div class=\\"ant-skeleton ant-skeleton-active\\"><div class=\\"ant-skeleton-content\\"><h3 class=\\"ant-skeleton-title\\" style=\\"width: 38%;\\"></h3><ul class=\\"ant-skeleton-paragraph\\"><li></li><li></li><li style=\\"width: 61%;\\"></li></ul></div></div></div></div></div></div></div><div data-index=\\"1\\" class=\\"slick-slide\\" tabindex=\\"-1\\" aria-hidden=\\"true\\" style=\\"outline: none; width: 0px;\\"><div><div tabindex=\\"-1\\" style=\\"width: 100%; display: inline-block;\\"><div style=\\"height: 600px;\\">2</div></div></div></div><div data-index=\\"2\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><div class=\\"listEventos\\" tabindex=\\"-1\\" style=\\"width: 100%; display: inline-block;\\"><div class=\\"list\\"><div class=\\"eventoCard\\"><div class=\\"ant-skeleton ant-skeleton-active\\"><div class=\\"ant-skeleton-content\\"><h3 class=\\"ant-skeleton-title\\" style=\\"width: 38%;\\"></h3><ul class=\\"ant-skeleton-paragraph\\"><li></li><li></li><li style=\\"width: 61%;\\"></li></ul></div></div></div><div class=\\"eventoCard\\"><div class=\\"ant-skeleton ant-skeleton-active\\"><div class=\\"ant-skeleton-content\\"><h3 class=\\"ant-skeleton-title\\" style=\\"width: 38%;\\"></h3><ul class=\\"ant-skeleton-paragraph\\"><li></li><li></li><li style=\\"width: 61%;\\"></li></ul></div></div></div><div class=\\"eventoCard\\"><div class=\\"ant-skeleton ant-skeleton-active\\"><div class=\\"ant-skeleton-content\\"><h3 class=\\"ant-skeleton-title\\" style=\\"width: 38%;\\"></h3><ul class=\\"ant-skeleton-paragraph\\"><li></li><li></li><li style=\\"width: 61%;\\"></li></ul></div></div></div><div class=\\"eventoCard\\"><div class=\\"ant-skeleton ant-skeleton-active\\"><div class=\\"ant-skeleton-content\\"><h3 class=\\"ant-skeleton-title\\" style=\\"width: 38%;\\"></h3><ul class=\\"ant-skeleton-paragraph\\"><li></li><li></li><li style=\\"width: 61%;\\"></li></ul></div></div></div></div></div></div></div><div data-index=\\"3\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><div tabindex=\\"-1\\" style=\\"width: 100%; display: inline-block;\\"><div style=\\"height: 600px;\\">2</div></div></div></div></div></div><ul style=\\"display: block;\\" class=\\"slick-dots slick-dots-top\\"><li class=\\"slick-active\\"><button>1</button></li><li class=\\"\\"><button>2</button></li></ul></div></div></div>"`;
